snakebite/client.py

```diff
@@ -40,6 +40,10 @@
 import errno
 import time
 import re
+import sys
+
+if sys.version > '3':
```

Comment:
This looks both a bit scary (since you're not using a library like six) and wrong (you meant  `>=` I guess? :)).